nbtc/relayer.go

```diff
@@ -102,6 +128,36 @@ func (r *Relayer) processPendingTxs() error {
 	return nil
 }
 
+// checkConfirmations checks all the broadcasted transactions to bitcoin
+// and if confirmed updates the database accordingly.
+func (r *Relayer) checkConfirmations() error {
+	broadcastedTxs, err := r.db.GetBroadcastedTxs()
+	if err != nil {
+		return err
+	}
+
+	for _, tx := range broadcastedTxs {
+		hash, err := chainhash.NewHash(tx.Hash)
+		if err != nil {
+			return err
+		}
+		txDetails, err := r.btcClient.GetTransaction(hash)
+		if err != nil {
+			return fmt.Errorf("error getting transaction details: %w", err)
+		}
+
+		// TODO: decide what threshold to use. Read that 6 is used on most of the cex'es etc.
+		if txDetails.Confirmations >= 6 {
```

Comment:
Can we set confirmation constant in parameters?

Reply:
Done